cli/commands/plan.go

```diff
@@ -57,14 +59,19 @@ func GetPlanName(cmd *PlanHandler) string {
 }
 
 func (cmd *PlanHandler) RunList(c *kingpin.ParseContext) error {
-	response := client.HTTPServiceGet("/v1/plans")
+	response := client.HTTPServiceGet("v1/plans")
 	client.PrintJSON(response)
 	return nil
 }
 
-func (cmd *PlanHandler) RunShow(c *kingpin.ParseContext) error {
-	response := client.HTTPServiceGet(fmt.Sprintf("v1/plans/%s", GetPlanName(cmd)))
-	client.PrintJSON(response)
+func (cmd *PlanHandler) RunStatus(c *kingpin.ParseContext) error {
+	planName := GetPlanName(cmd)
+	response := client.HTTPServiceGet(fmt.Sprintf("v1/plans/%s", planName))
+	if cmd.RawJson {
+		client.PrintJSON(response)
+	} else {
+		client.LogMessage(toStatusTree(planName, client.GetResponseBytes(response)))
```

Comment:
Same as for `update.go`, we should use client.PrintMessage here to avoid printing the timestamp.